modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/utils/clients/APIStoreRestClient.java

```diff
@@ -1271,6 +1271,27 @@ public HttpResponse signUp(String userName, String password, String firstName, S
             throw new APIManagerIntegrationTestException("Error in user sign up. Error: " + e.getMessage(), e);
         }
     }
+    /**
+     * API Store sign up
+     *
+     * @param userName - store user name
+     * @param password -store password
```

Comment:
slight issue with - need to have a space

Reply:
fixed